unit_tests/test_base_version.py

```diff
@@ -14,7 +14,8 @@
 
 import unittest
 
-from utils.get_supported_scylla_base_versions import UpgradeBaseVersion  # pylint: disable=no-name-in-module
+from utils.get_supported_scylla_base_versions import UpgradeBaseVersion \
```

Comment:
why on new line?

Reply:
Pycharm showed that it was over the limit, but it turned out that it is not/